### DISCUSSION TITLE
rule "flippinCactusEnhanced"

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#126] [Ideas]
Adds more rotateable blocks to the list.
- Chest
- Barrels
- Slabs (Moving up/down)
- Hopper
- Shulker Boxes

Also allowing shift+right click flippin for some blocks.

Just a few blocks I came up with right now 😄

**Comment by wendavid552** [comment]
Actually Carpet mod could already flip:
- Hopper
- Slabs

For the rest containers, perhaps you need a shift rotation? or you can use a debug stick?
